### PR TITLE
feat(scan): Sound FX

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
Adds three sound effects to VxScan: success, warning, and failure. Plays them on mount of the appropriate screens.

## Testing Plan
Manual testing

### CODE REVIEW

**Comment by mattroe** [comment]
per @benadida, we're going to need to make sure system volume on vxscan is set

**Review by benadida — APPROVED** [review]

File: frontends/precinct-scanner/src/hooks/use_sound.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import useSoundLib from 'use-sound';
+
+export function useSound(sound: 'success' | 'warning' | 'error'): () => void {
+  const [playSound] = useSoundLib(`/sounds/${sound}.mp3`);
```
Comment (arsalansufi) [inline frontends/precinct-scanner/src/hooks/use_sound.ts:4]: Cool library!

**Review by arsalansufi — APPROVED** [review]
The error sounds made me jump and really caught my attention at full volume on my headphones 😆 - probably a good thing that it's so attention grabbing!

File: frontends/precinct-scanner/src/hooks/use_sound.ts (hunk 1 above)
Comment (jonahkagan) [reply frontends/precinct-scanner/src/hooks/use_sound.ts:4]: I know! So easy!

**Comment by jonahkagan** [comment]
@arsalansufi lol, sorry not sorry

**Pull request merged**